content/nginx-one/secure-your-fleet/secure.md

```diff
@@ -0,0 +1,131 @@
+
+With access, you or someone in your organization should have an email from no-reply@cloud.f5.com asking you to update your password when the tenant was created. The account name referenced in the E-Mail in bold is the tenant name.
+
+Navigate to https://INSERT_YOUR_TENANT_NAME.console.ves.volterra.io/ to access F5 Distributed Cloud. If you have never logged in, select the **Forgot Password?** option in the log in screen. Alternatively, if someone within your organization has access, ask them to add you as a user within your tenant with a role providing permissions for NGINX One.
+
+### Confirm access to NGINX One Console in the F5 Distributed Cloud
+
+Once you've logged in with your password, you should be able to see and select the NGINX One tile. 
+
+1. Select the **NGINX One** tile<!-- Style note: this is the UI in DC -->
+1. Select **Visit Service**
+
+### Install an instance of NGINX
+
+Ensure you have an instance of [NGINX Open Source or NGINX Plus]({{< ref "/nginx/admin-guide/installing-nginx/" >}}) installed and available. This guide provides instructions for connecting an instance installed in a Linux environment (VM or bare metal hardware) where you have command line access.
+Alternatively, we also have instructions for [Deploying NGINX and NGINX Plus with Docker]({{< ref "/nginx/admin-guide/installing-nginx/installing-nginx-docker.md" >}}) with NGINX and the NGINX Agent installed. That deployment can connect with environment variables.
+
+## Connect at least one NGINX instance to the NGINX One Console
+
+If you already have connected instances to the NGINX One Console, you can start to [Configure an active alert policy]({{< ref "/nginx-one/secure-your-fleet/secure.md#configure-an-active-alert-policy" >}}).
+Otherwise, you need to generate a data plane key, add an instance, and install NGINX Agent. We assume this is the first time you are connecting an instance.
+
+### Add an instance
+
+{{< include "/nginx-one/how-to/add-instance.md" >}}
+
+### Generate a data plane key
+
+{{< include "/nginx-one/how-to/generate-data-plane-key.md" >}}
+
+### Install NGINX Agent
+
+{{< include "/nginx-one/how-to/install-nginx-agent.md" >}}
+
+You can also install NGINX Agent from our repositories and configure it manually. Alternatively you can use our official NGINX Docker images, pre-configured with NGINX Agent.
+
+## Configure an active alert policy
+
+The NGINX One Console monitors all connected NGINX instances for CVEs and insecure configurations. Using the F5 Distributed Cloud's Alert Policies you can receive alerts for these risks in a manner of your choosing; for the purposes of this guide we will show you how to configure E-Mail alerts.
```

Comment:
```suggestion
The NGINX One Console monitors all connected NGINX instances for CVEs and insecure configurations. Using the F5 Distributed Cloud's Alert Policies, you can receive alerts for these risks in a manner of your choosing; for the purposes of this guide, we will show you how to configure E-Mail alerts.
```

Reply:
Style guide says to use email: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/email

Also, use present tense predominately.